crates/eframe/src/native/run.rs

```diff
@@ -160,7 +162,7 @@ fn run_and_return(
             EventResult::Exit => {
                 log::debug!("Asking to exit event loop…");
                 winit_app.save_and_destroy();
-                *control_flow = ControlFlow::Exit;
+                event_loop_window_target.exit();
```

Comment:
Do we still need this `winit_app.save_and_destroy()` - we are going to get a `Event::LoopExiting` event later?`